package.json

```diff
@@ -6,7 +6,17 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "type-check": "tsc --noEmit",
+    "lint": "eslint --ignore-path .gitignore \"src/**/*.+(ts|js|tsx)\"",
+    "format": "prettier --ignore-path .gitignore \"src/**/*.+(ts|js|tsx)\" --write",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "./src/**/*.{ts,js,jsx,tsx}": [
+      "eslint --ignore-path .gitignore --fix",
+      "prettier --ignore-path .gitignore --write"
+    ]
```

Comment:
can we setup a CI script to run all these?

Reply:
I used this in the pre commit script with Husky so we have everything linted and formatted even before it gets commited so I don't see the point of having this in CI because it no errors or warning can't get there.

Reply:
can ppl not get around that in funky ways? feels like it wouldnt hurt to double check

Reply:
Hmm I don't think that it is possible